extension/fts/src/function/query_fts_bind_data.cpp

```diff
@@ -10,6 +10,9 @@
 #include "storage/storage_manager.h"
 #include "storage/table/node_table.h"
 #include "utils/fts_utils.h"
+#if defined(KUZU_ENABLE_JIEBA)
```

Comment:
I think we don't need this option, we can enable chinese tokenizer support by default.

Reply:
Yeah indeed, sorry!